apps/docs/pusher.md

```diff
@@ -1,16 +1,4 @@
-# Stork CLI
-
-A suite of tools to interact with Stork's services and on-chain contracts.
-
-## evm-push
-
-### Background
-
-Stork signed data feeds are delivered off-chain from publishers to subscribers via Stork's aggregation network. In order for this data to be usable on-chain, it must be written to the Stork contract on any EVM compatible network. This tool is used to push signed data feeds to the Stork contract.
-
-Stork does not write this data to the chain directly by default, but any subscriber can choose to write the data to the chain if they so choose. This tool can be used to facilitate that process.
-
-### Usage
+## Usage
```

Comment:
should probably still have some kinda top level header like `# Pusher` or `# Chain Pusher`